Localization/es-419.lproj/Localizable.strings

```diff
@@ -161,7 +161,7 @@
 "Password" = "Password";
 "Cancel boarding?" = "Cancel boarding?";
 "This boarding will not be saved" = "This boarding will not be saved";
-"Go back" = "Go back";
+"Cancel Boarding" = "Cancel Boarding";
 "Log In" = "Log In";
```

Comment:
For now we have key "Cancel boarding?" = "Cancel boarding?"; (line 162). Is it really needed to create new key without quotation mark?